src/components/MovieList/MovieList.jsx

```diff
@@ -1,34 +1,46 @@
-import React, {Component} from 'react';
-import Axios from 'axios';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import API from '../../api';
 import MovieItem from '../MovieItem/MovieItem';
-import styles from './movie-list.scss';
+import styles from './MovieList.scss';
+
+const cx = classNames.bind(styles);
 
 class MovieList extends Component {
   constructor() {
     super();
 
     this.state = {
-      movies: []
+      movies: [],
     };
+  }
 
-    MovieList.getMovies().then(movies => {
+  componentDidMount() {
+    API.getMovies().then((movies) => {
       this.setState({
-        movies: movies.data.map(movie => {
-          return <MovieItem key={movie.id} title={movie.title} description={movie.description} />;
-        })
-      })
+        movies: [...movies.data],
+      });
     });
-
-  }
-
-  static getMovies() {
-    return Axios.get('http://localhost:3000/api/v1/movies');
   }
 
   render() {
+    const { movies } = this.state;
     return (
-      <div className={styles.container}>
-        { this.state.movies.length > 0 ?  this.state.movies : (<h1>No movies found</h1>)}
+      <div className={cx(styles.container, styles.clearfix)}>
+        { movies.length > 0 ? (
```

Comment:
There is no need to check length. If it will be 0 then map will not be called and nothing will be rendered

Reply:
I am checking length to render message `No movies found` if it's 0.